ecl/hql/hqlerror.cpp

```diff
@@ -206,7 +206,7 @@ class HQL_API FileErrorReceiver : public ErrorReceiverSink
 
         StringBuffer msg;
         error->errorMessage(msg);
-        if (!filename) filename = isError(severity) ? "" : *unknownAtom;
+        if (!filename) filename = isError(severity) ? "" : str(unknownAtom);
```

Comment:
I won't point any more out unless they are really critical.  Not really necessary to change any of them.